docs/labwc-config.5.scd

```diff
@@ -177,6 +177,7 @@ this is for compatibility with Openbox.
   <autoEnableOutputs>yes</autoEnableOutputs>
   <reuseOutputMode>no</reuseOutputMode>
   <xwaylandPersistence>no</xwaylandPersistence>
+  <primarySelection>yes</primarySelection>
 </core>
```

Comment:
Misses the same in `rc.xml.all`

Reply:
Thanks. Good catch. Can't believe I forgot that!